docs/docs/users/funding.md

```
@@ -0,0 +1,15 @@
+sidebar_position: 7
+---
+
+<p align="center" style={{ display: 'flex' , justifyContent: 'space-around' }}>
+  <img src="/img/logo.png" alt="Forest logo"/>
+  <img src="/img/chainsafe_logo.png" alt="ChainSafe logo"/>
+  <img src="/img/filecoin_logo.png" alt="Filecoin logo"/>
+</p>
+
+Forest project is funded by the [Filecoin Foundation](https://fil.org/) and developed by [ChainSafe Systems](https://chainsafe.io). The project has also received funding from the [FIL-RetroPGF](https://www.fil-retropgf.io/) Retroactive Public Goods Funding rounds.
+
+Hypercerts, which prove the public goods contributions of Forest to the Filecoin ecosystem, are available [here](https://app.hypercerts.org/profile/0xb4713f39476841fAF0ea5a555d0b1d451e6B05A1).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)